recipes/workalendar/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "workalendar" %}
+{% set version = "5.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3580a49fc7953629abcdd7219c7a7ac7437b48159b8f65c2fa580654077e5dd5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - ephem
```

Comment:
Do you need all of these? Often `python` and `pip` is enough.

Reply:
Well, I saw them listed on the project’s `setup.py` so I thought not adding them was the problem, but apparently not

Reply:
The `host` dependencies should be as minimal as possible. Occasionally packages import everything for no good reason but at a glance of the `setup.py` I think you can simplify it down to just three.

Reply:
Oh wait, I thought we were talking about the ray package from the other PR, not this one lol. I built this file with `conda skeleton` so I just assumed it knew what it was doing

Reply:
Looking at the [setup.py](https://github.com/peopledoc/workalendar/blob/master/setup.pyl), these dependencies are listed:
`'ephem',
    'python-dateutil',
    'lunardate',
    'pytz',
    'pyCalverter',
    'pyluach',
    'setuptools>=1.0'`

Reply:
`host` only needs to contain the dependencies required to run the `setup.py` script (in setuptools speak, `setup_requires` rather than `install_requires`).

For figuring out the host dependencies you can often start minimal (`python` and `pip`) and then add packages until it passes.

Reply:
Just tried building it locally without host dependencies and it worked! That's very enlightening for future builds.